access_requests_test.go

```diff
@@ -11,7 +11,10 @@ import (
 )
 
 func TestListProjectAccessRequests(t *testing.T) {
-	mux, server, client := setup()
+	mux, server, client, err := setup()
```

Comment:
Wouldn't it be better to just pass `t` here and do a `t.Fatalf()` inside to unwind the stack without returning an error and changing the signature?

Reply:
I think it's better, but mainly because it saves a few lines of code. The signature will still change of course 😏 But will change this one...